zerver/views/message_send.py

```diff
@@ -211,6 +211,11 @@ def send_message_backend(
     time: Optional[float] = REQ(default=None, converter=to_float, documentation_pending=True),
 ) -> HttpResponse:
     message_type = message_type_name
+    if message_type == "direct":
+        # For now, use "private" message type for direct messages.
+        # TODO: Use "direct" type here, as well as in events and
+        # message (created, schdeduled, drafts) objects/dicts.
```

Comment:
A TODO here sounds reasonable. I think changing events is probably out of scope for this release; it's a more complex migration requiring a client capability.

Reply:
Yeah, that makes sense. Mostly trying to leave us in a good place with these changes for looking at that more complex change for after the release settles.